packages/plugin-typescript/package.json

```diff
@@ -5,7 +5,8 @@
   "dependencies": {
     "@berry/builder": "workspace:*",
     "@berry/core": "workspace:*",
-    "@berry/plugin-essentials": "workspace:*"
+    "@berry/plugin-essentials": "workspace:*",
+    "@berry/plugin-pack": "workspace:*"
```

Comment:
I'm not sure I like having to depend on plugin-pack just for a typescript interface definition

Reply:
It's a bit different from standard bundling strategies - the `@berry/builder` toolchain doesn't actually aggrgate the plugins together when building one of them; they're instead expected to be downloaded independently by the user - or in the case of `plugin-pack`, directly linked against the version of `plugin-pack` bundled within the standard bundle (it's a bit like if they were peer dependencies, in a sense).